src/Components/Movie/styles.scss

```diff
@@ -0,0 +1,45 @@
+.movie-card {
+    margin-bottom: 50px;
+    flex: 0 1 30%;
+    display: flex;
+    flex-wrap: wrap;
+    color: #ffff;
```

Comment:
shouldn't it be `#fff`?

Reply:
Yeah... you right! I have copied this value from scrapbook
Will change